src/tinybox.vue

```diff
@@ -202,6 +202,38 @@ export default {
         window.removeEventListener('keyup', this.keyup);
       }
     },
+    /*
+     * Center the thumbnails' scrollbar to the clicked image
+     */
+    index(index) {
+      let elements;
+      let el;
+      let thumbs;
```

Comment:
Since these are not used outside the `if` statement, you can merge declarations and assignments